script/setup/critools-version

```diff
@@ -1 +1 @@
-53ad8bb7f97e1b1d1c0c0634e43a3c2b8b07b718
+1a0b272405a694d7d82606c5a7eddd7905321c4b
```

Comment:
hate to do this but can we also try critest v1.24.0 and go back to the 1.16.5 ginkgo and with the removal of the parallel from skip for vagrant?

Reply:
okily-dokily